mlserver/parallel/dispatcher.py

```diff
@@ -93,7 +106,19 @@ def _get_worker(self) -> Worker:
         By default, this is just a round-robin through all the workers.
         """
         worker_pid = next(self._workers_round_robin)
-        return self._workers[worker_pid]
+        return self._workers[worker_pid], worker_pid
+    
+    def _workers_queue_monitor(self, worker: Worker, worker_pid: int):
+        """Get metrics from every worker request queue"""
+        queue_size = worker._requests.qsize()
+        
+        queue_request_count.labels(workerpid=str(worker_pid)).observe(
+            float(queue_size)
+        )
+    
+    def _workers_processes_monitor(self, loop: AbstractEventLoop):
+        process_request_count.observe(float(len(asyncio.all_tasks(loop))))
```

Comment:
This `loop` would only represent the main process' loop. So it wouldn't capture the workers' queued asyncio tasks. 

For this first iteration, I'd keep it simple and focus just on tracking the queue.

Reply:
So I will focus on only in queue requests in this PR. I will do a couple of tests.

Reply:
Regarding to the tests, I noticed you mount an aiohttp server in a fixture however I'd need to test the metrics inside the code in mlserver folder and I don't know how to fit real metrics in a mocked server.

Reply:
Wouldn't it be the same as in the current metrics tests? Those spin up an actual MLServer instance, send a couple requests, and then scrape the metrics endpoint to compare the metrics before and after (and assert that they've increased).